Frontend/ui-customer/src/Flow.purs

```diff
@@ -241,10 +241,34 @@ currentFlowStatus = do
       in
         void $ lift $ lift $ Remote.updateProfile (UpdateProfileReq requiredData)
     
-    updateUserLanguage :: Maybe String -> FlowBT String Unit
-    updateUserLanguage language = 
-      when (isNothing language || (getKeyByLanguage (fromMaybe "ENGLISH" language) /= (getLanguageLocale languageKey)))
-        $ void $ lift $ lift $ Remote.updateProfile (Remote.mkUpdateProfileRequest FunctionCall)
+    updateLanguageAndReferralCode :: Maybe String -> FlowBT String Unit
+    updateLanguageAndReferralCode language = do
+      config <- getAppConfigFlowBT appConfig
+      let referralStatus = getValueToLocalStore REFERRAL_STATUS
```

Comment:
Can you mention the usage of this local storage variable and
What all values it can have ?